monitoring/security/securitystatus.py

```diff
@@ -1,22 +1,33 @@
 #!/usr/bin/env python3
 
-import json
-import subprocess
+# Prints the number of ESM updates that the instance can see, but not
+# install because it does not have the relevant ESM service enabled.
+# This helps administrators to identify and prioritise the enablement
+# of ESM across a fleet.
+import os
 import sys
 
-return_info = subprocess.check_output(["ua",  "security-status", "--format=json"])
-json_output = return_info.decode(sys.stdout.encoding)
-ua_security_status = json.loads(json_output)
+try:
+    from uaclient.api.u.pro.packages.updates.v1 import updates
+    from uaclient.api.u.pro.status.enabled_services.v1 import enabled_services
 
-if "esm-infra" not in ua_security_status["summary"]["ua"]["enabled_services"]:
-    unpatchable_esm_infra = ua_security_status["summary"]["num_esm_infra_updates"]
-else:    
-    unpatchable_esm_infra = 0
+    # Some of these calls to the pro api can take time and we have a 10 second limit on Landscape scripts
+    # so we will call the endpoints once and reuse the results
+    update_information_summary = updates().summary
+    enabled_pro_services_list = list(i.name for i in enabled_services().enabled_services)
 
-if "esm-apps" not in ua_security_status["summary"]["ua"]["enabled_services"]:
-    unpatchable_esm_apps = ua_security_status["summary"]["num_esm_apps_updates"]
-else:
-    unpatchable_esm_apps = 0    
+    unpatchable_esm_updates = 0
 
-total_unpatchable_esm_updates = unpatchable_esm_infra + unpatchable_esm_apps
-print(total_unpatchable_esm_updates)
\ No newline at end of file
+    if ("esm-infra" not in enabled_pro_services_list) and (update_information_summary.num_esm_infra_updates > 0):
+        unpatchable_esm_updates += update_information_summary.num_esm_infra_updates
+
+    if ("esm-apps" not in enabled_pro_services_list) and (update_information_summary.num_esm_apps_updates > 0):
+        unpatchable_esm_updates += update_information_summary.num_esm_apps_updates
+
+    print(unpatchable_esm_updates)
+    sys.exit(os.EX_OK)
+
+except ImportError:
```

Comment:
nit: I'd prefer the `except` closer to the code that would raise `ImportError` (L11-12), but this is a short enough script that I'm not too bothered.